### PR TITLE
Fixed auth URIS.

### PR DESCRIPTION
**Pull request by margawron**

### CODE REVIEW

**Review by syphzur — APPROVED** [review]

File: PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml [hunk 1]
```diff
@@ -31,85 +31,85 @@ Proper file format example:
...
+                <access>/orderItem/single</access>
+                <access>/orderItem/patch</access>
+                <access>/orderItem/delete</access>
+                <access>/orderItem/post</access>
 
-                <access>picture/single</access>
-                <access>picture/patch</access>
-                <access>picture/delete</access>
-                <access>picture/post</access>
+                <access>/picture/single</access>
```
Comment (syphzur) [inline PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml]: /picture/single url is not protected by security

**Review by syphzur — CHANGES_REQUESTED** [review]
delete public urls from admin role

File: PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml (hunk 1 above)
Comment (margawron) [reply PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml]: Check bf0ba94

File: PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml [hunk 2]
```diff
@@ -31,85 +31,81 @@ Proper file format example:
...
-                <access>address/patch</access>
-                <access>address/delete</access>
-                <access>address/post</access>
+                <access>/address/all</access>
+                <access>/address/single</access>
+                <access>/address/patch</access>
+                <access>/address/delete</access>
+                <access>/address/post</access>
 
-                <access>category/all</access>
```
Comment (latusikl) [inline PhotoPlusBackend/photoplus/src/main/resources/roleConfig.xml:52]: You should add endpoints which have to be accessible to all to role all.

**Review by latusikl — CHANGES_REQUESTED** [review]

**Review by syphzur — APPROVED** [review]

**Review by syphzur — APPROVED** [review]

**Review by latusikl — APPROVED** [review]

**Pull request merged**